services/forest/src/persistence/SCIHF.js

```diff
@@ -0,0 +1,538 @@
+module.exports = (
+  db,
+  { geoIntegrity },
+  logger,
+) => ({
+  /**
+   * Find the area grouped by SCI, HF persistence and PA categories
+   * in the given environmental authority
+   *
+   * @param {String} areaId environmental authority id
+   * @param {Number} year optional year to filter data, 2018 by default
+   *
+   * @returns {Object[]} Array of areas grouped by SCI, HF persistence and PA categories
+   */
+  findSCIHFInEA: async (areaId, year = 2018) => {
+    try {
+      return geoIntegrity.query()
+        .where({ id_ea: areaId, sci_year: year })
+        .select('hf_pers', 'sci_cat', 'binary_protected')
+        .sum('area_ha as area')
+        .groupBy('binary_protected', 'sci_cat', 'hf_pers')
+        .orderBy('binary_protected', 'sci_cat', 'hf_pers');
+    } catch (e) {
+      logger.error(e.stack || e.Error || e.message || e);
+      throw new Error('Error at findSCIHFInEA persistence');
```

Comment:
This will be the de user message, I suggest something less specific, the user doesn't need to know there was an error in the persistence

Reply:
And the same for the other functions

Reply:
Done